api/v1/services/AuthService.js

```diff
@@ -69,7 +69,13 @@ module.exports.verify = (token) => _Promise.try(() =>
     throw new errors.UnprocessableRequestError(message);
   });
 
-module.exports.getGitHubSessionCodeURL = () => GITHUB_AUTH_REDIRECT + config.auth.github.id;
+module.exports.getGitHubSessionCodeURL = (isMobile) => {
+  if (!_.isUndefined(isMobile)) {
+    return GITHUB_AUTH_REDIRECT + config.auth.github.id + '&redirect_uri=' + config.auth.github.mobileRedirect;
```

Comment:
I'll note again the intention of this for posterity: `redirect_uri` exists here so that GitHub redirects the user to a special mobile handler route on the front-end, which in turn redirects to a deep link (instead of just continuing with sign-in). 

This helps to resolve some compilcations with authentication. It might be expected that the mobile client can just catch the redirect URL with a universal link of some sort, but certain platforms (i.e. iOS) frustratingly jumble this process.